services/s3/src/test/java/software/amazon/awssdk/services/s3/internal/crossregion/S3CrossRegionAsyncClientTest.java

```diff
@@ -98,6 +98,13 @@ void crossRegionClient_createdWithWrapping_SuccessfullyIntercepts() {
         assertThat(captureInterceptor.endpointProvider).isInstanceOf(S3CrossRegionAsyncClient.BucketEndpointProvider.class);
     }
 
+    @Test
+    void standardOp_crossRegionClient_containUserAgent() {
```

Comment:
can we also test that the user agent contains something that was there before the change?

Reply:
Done